bin/start.js

```diff
@@ -1,11 +1,15 @@
 #!/usr/bin/env node
 
+var log = require('npmlog')
+log.level = 'warn'
+
 var cli = require('../cli')
 
 var emoji = require('node-emoji')
 
 cli(function (error, server) {
   if (error) {
+    log.warn('DB url config', 'If you are using --dbUrl, --dbUrlUsername, --dbUrlPassword, you should provide your authentication details without URL encoding')
```

Comment:
My guess is that it should go directly to the place where we parse the URL. If the parsed URL has no authentication, then we should log an explanation with `log.error`